src/plugin.js

```diff
@@ -86,7 +87,7 @@ export class ${className} implements SkyLibResourcesProvider {
   private resources: any = ${JSON.stringify(resources)};
 
   public getString(localeInfo: SkyAppLocaleInfo, name: string): string {
-    const locale = localeInfo.locale.replace('-', '_');
+    const locale = localeInfo.locale.toLowerCase().replace('-', '_');
```

Comment:
Good catch! Needs a unit test.

Reply:
Tests have been updated to check for casing.